packages/python/m/cli/commands/github/actions.py

```diff
@@ -0,0 +1,50 @@
+from m.cli import Arg, BaseModel, command, run_main
+
+
+class Arguments(BaseModel):
+    """Create the actions metadata as described by given python script.
+
+    When creating actions we need to have the `python_file` be at the root
+    of a python project. This is because when we run the action we will set
+    the path to the directory containing the file.
+
+    By default it has been set to `src/actions.py` since this common case for
+    python projects.
+
+    The `actions.py` can be name anything but it must contain a variable named
+    `actions`. This variable must be an instance or list of
+    `m.github.actions.Action`.
+    """
+
+    python_file: str = Arg(
+        default='src/actions.py',
+        help='python script containing the `actions` variable',
+        positional=True,
+    )
```

Comment:
This is a prime candidate for a [validator](https://docs.pydantic.dev/2.4/api/functional_validators/#pydantic.functional_validators.model_validator)

Reply:
```
$ m github actions bad/path/to/file.py
usage: m github actions [-h] [--check] [--show-traceback] [python_file]
m github actions: error: argument python_file: file does not exist
```